### PR TITLE
Table format

### PR DESCRIPTION
**Pull request by sandertyu**
This pull request fulfills everything outlined in issue #46 about touching up the datatable. I have opened a seperate issue #47 for displaying the steer axis tilt in degrees rather than radians. Everything here is ready to be merged.

### CODE REVIEW

**Comment by sandertyu** [comment]
The datatable now correctly displays steer axis tilt in degrees, and bicycleparameters calculates it in radians. This push is totally set to go.

File: bicycleparameters/cycle_app.py [hunk 1]
```diff
@@ -197,6 +198,13 @@ def populate_wheel_data(value, n_clicks):
         genLabels.append({'label': i})
     for i in range(8, 12):
         con.append({'con': genPure.get(pList[i])})
+
+    # converts radians to degrees for display in the table
+    lamD = con[2]
+    radians = lamD.get('con')
+    degrees = (radians*180)/math.pi
+    con[2] = {'con': int(degrees)}
```
Comment (moorepants) [inline bicycleparameters/cycle_app.py:206]: NumPy has the rad2deg and deg2rad functions that you may want to use in the future.

**Comment by moorepants** [comment]
Just tested this out. I'd looking good and runs fast now. Seems like you fixed the issue about executing the callback too many times.

**Comment by moorepants** [comment]
I take that back. Changing the data in the table results in a single callback to update the plot but changing bikes still executes like 7 times.

**Pull request merged**